In [1]:
import pandas as pd
import requests
from bs4 import BeautifulSoup

In [6]:
response = requests.get("https://www.ambitionbox.com/list-of-companies?campaign=desktop_nav")

In [8]:
response

<Response [403]>

In [ ]:
response.text

'<HTML><HEAD>\n<TITLE>Access Denied</TITLE>\n</HEAD><BODY>\n<H1>Access Denied</H1>\n \nYou don\'t have permission to access "http&#58;&#47;&#47;www&#46;ambitionbox&#46;com&#47;list&#45;of&#45;companies&#63;" on this server.<P>\nReference&#32;&#35;18&#46;7a91160&#46;1760023751&#46;21e43dc9\n<P>https&#58;&#47;&#47;errors&#46;edgesuite&#46;net&#47;18&#46;7a91160&#46;1760023751&#46;21e43dc9</P>\n</BODY>\n</HTML>\n'

It is showing error [403] means bad request the data can't be fetch due to access denied

In [16]:
headers={'User-Agent':'Mozilla/5.0 (Windows NT 6.3; Win 64 ; x64) Apple WeKit /537.36(KHTML , like Gecko) Chrome/80.0.3987.162 Safari/537.36'}
webpage = requests.get('https://www.ambitionbox.com/list-of-companies?campaign=desktop_nav&page=1',headers=headers).text